src/PostgREST/SchemaCache/Routine.hs

```diff
@@ -69,10 +69,11 @@ instance JSON.ToJSON Routine where
     ]
 
 data RoutineParam = RoutineParam
-  { ppName :: Text
-  , ppType :: Text
-  , ppReq  :: Bool
-  , ppVar  :: Bool
+  { ppName          :: Text
+  , ppType          :: Text
+  , ppTypeMaxLength :: Text
```

Comment:
Is `ppType` now unused? Would it make sense to have a single `ppType` with the added length for these cases?

Reply:
No, `ppType` is still used on every other place (e.g. errors, OpenAPI, etc.) Only when building the Query it transforms e.g. `character` -> `character varying`.

The thing is, the length is ignored by PostgreSQL, it's not specified in the function definition, so no length can be added, it's the `character` -> `character varying` change that's done here.